### PR TITLE
Ticker suggestion

### PR DESCRIPTION
**Pull request by gorilazish**
Dar pri modalo reikes padirbet:
- Dizains
- Input validation / OK button disable
- Galimybe prisidet coina jeigu neivedi Buy Amount ir Buy Price

### CODE REVIEW

File: src/app/containers/CreateNewItemView/CreateNewItemView.tsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import React from 'react'
+import Autocomplete from 'react-autocomplete'
 import { RouteComponentProps } from 'react-router-dom'
+import { connect, FelaWithStylesProps } from 'react-fela'
 import { observer, inject } from 'mobx-react'
 import { Flex, Box } from 'reflexbox'
-import { Input } from '../../components'
-import { RootStore } from '../../stores/RootStore';
+import { Button, Input } from '../../components'
+import { RootStore } from '../../stores/RootStore'
```
Comment (martynasj) [inline src/app/containers/CreateNewItemView/CreateNewItemView.tsx]: jei vien del typu, tai nebutina situ importinti, as anus esu padaris ant global scope. Nzn ar good practice ar ne :D

File: src/app/containers/CreateNewItemView/CreateNewItemView.tsx [hunk 2]
```diff
@@ -137,43 +189,76 @@ class CreateNewItemView extends React.Component<IProps, IState> {
     return this.props.portfolioStore!.getItem(this.props.match.params.id)
   }
 
+  private filterSymbolSuggestions = (item: TickerModel, value: string) => {
+    const tickerName = item.name.toLowerCase()
```
Comment (martynasj) [inline src/app/containers/CreateNewItemView/CreateNewItemView.tsx:192]: value irgi reiketu toLowercase

File: src/app/containers/Toolbar/Toolbar.tsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { Flex } from 'reflexbox'
 import { inject, observer } from 'mobx-react'
+import { Button } from '../../components'
+import { theme } from '../../theme'
```
Comment (martynasj) [inline src/app/containers/Toolbar/Toolbar.tsx:5]: reiketu gal ton theme injectinti per Provider. Tada galetumem turet galimybi pasikeisti tema runtime. pvz dark ir light :) bet cia kitam kartui

Comment (gorilazish) [reply src/app/containers/Toolbar/Toolbar.tsx:5]: good idea

**Review by martynasj — APPROVED** [review]

**Pull request merged**